variables.tf

```diff
@@ -204,6 +204,16 @@ variable "update_dsi_spreadsheets_task_schedule" {
   default     = "cron(0 02 * * ? *)"
 }
 
+variable "update_database_records_in_big_query_task_command" {
```

Comment:
I'm not sure I understand what value having this indirection here and below adds. 🤔

I know the other tasks do it, but I'm not sure _why_.